docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  mysql:
+    image: mysql:5.7
+    container_name: test-mysql
+    environment:
+      - DATABASE_NAME=test_db
+      - DATABASE_PASSWORD=password
```

```suggestion
      - MYSQL_DATABASE=test_db
      - MYSQL_ROOT_PASSWORD=password
```

このように書く事が一般的です。
元々の記載でも動きましたでしょうか？

---

ご指摘ありがとうございます。
確認したところ正常に動作していなかったため、
記載頂いた内容で修正するように致します。